feat: jwt folder

bart-krakowski opened the pull request:
This PR introduces the icon for the folder of [JWT](https://jwt.io/).
![Frame 23](https://user-images.githubusercontent.com/48633090/173548218-f3568fd9-ffdd-490d-8b3e-8380e753ed06.png)

Closes: #1351

ultram4rine commented:
Recursion detected! I think you mean #1351 @bart-krakowski

PKief reviewed (commented):
Thanks for your PR, however I find the icon on the folder icon too colorful. I usually try to make the icons that are on the folder icons one color. They have the same color as the folder icon behind, just a bit lighter. Only in very rare cases I make the icon colored, but only if the color absolutely serves as recognition value. In this case, however, I don't feel that way. Furthermore, it would be good if the icon was not quite so far below the folder icon. It would have to be moved up a bit. I often use this folder icon as a reference: https://github.com/PKief/vscode-material-icon-theme/blob/main/icons/folder-dist.svg.

I also usually try to make the folder colors not stand out as much as file icons. Especially with the jwt-folder-icon it is so that it does not have to stand out incredibly strong, because a jwt is now also not so super important as that it must be so strongly emphasized in the entirety of a project directory.

bart-krakowski commented:
Thanks for your input! I changed colors. What do you think about it now?

![jwt](https://user-images.githubusercontent.com/48633090/177001489-9d4f2968-bca3-4df9-902c-69d86cea517f.png)

PKief commented:
> Thanks for your input! I changed colors. What do you think about it now?
> 
> ![jwt](https://user-images.githubusercontent.com/48633090/177001489-9d4f2968-bca3-4df9-902c-69d86cea517f.png)

Thanks it looks better. Still I think the JWT logo is not a good fit for jwt folder icons. What do you think if we just associate jwt folder with keys folder like shown here:

![image](https://user-images.githubusercontent.com/12248527/177207655-10c49942-1de9-4466-9e30-dc38277f4a08.png)

Or maybe also with the tag-icon:

https://materialdesignicons.com/icon/tag

But I'd be fine with the keys folder icon which already exists. So we'd just need to associate it with the "jwt" folder name 🙈

bart-krakowski commented:
> > Thanks for your input! I changed colors. What do you think about it now?
> > ![jwt](https://user-images.githubusercontent.com/48633090/177001489-9d4f2968-bca3-4df9-902c-69d86cea517f.png)
> 
> Thanks it looks better. Still I think the JWT logo is not a good fit for jwt folder icons. What do you think if we just associate jwt folder with keys folder like shown here:
> 
> ![image](https://user-images.githubusercontent.com/12248527/177207655-10c49942-1de9-4466-9e30-dc38277f4a08.png)
> 
> Or maybe also with the tag-icon:
> 
> https://materialdesignicons.com/icon/tag
> 
> But I'd be fine with the keys folder icon which already exists. So we'd just need to associate it with the "jwt" folder name 🙈

@PKief, I've removed the JWT icon and associated it with the `folder-keys` icon.
